app/assets/stylesheets/search.scss

```diff
@@ -1,39 +1,119 @@
+$color: #566473;
+
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #888;
+}
+
 #search-form {
+  margin-right: 10px;
   position: relative;
 
-  #search {
-    width: 260px;
+  .search-box {
+
+    &::after,
+    &::before {
```

Comment:
the same styles, than at app/assets/stylesheets/header.scss, can you make them reusable?